### PR TITLE
gpxsee: 11.6 -> 11.8

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Description of changes

Add support for building with qt6 since gpxsee looks foreign with qt5 under KDE
anyway, we might as well start with qt6 that looks equally foreign.

I don't have access to anything other than linux x86_64 for testing, so leave
the qt5 option in here for now.

Changelog: https://build.opensuse.org/package/view_file/home:tumic:GPXSee/gpxsee/gpxsee.changes

Does it work? If I don't get lost this weekend while MTB'ing, it should be fine.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sikmir** [comment]
Result of `nixpkgs-review pr 201440` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>gpxsee-qt6</li>
    <li>qt6Packages.qtpbfimageplugin</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gpxsee (gpxsee-qt5)</li>
  </ul>
</details>

**Comment by peterhoeg** [comment]
@GrahamcOfBorg build gpxsee-qt5 gpxsee-qt6

**Comment by rapenne-s** [comment]
Result of `nixpkgs-review pr 201440` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>gpxsee (gpxsee-qt5)</li>
    <li>gpxsee-qt6</li>
    <li>qt6Packages.qtpbfimageplugin</li>
  </ul>
</details>

**Review by rapenne-s — APPROVED** [review]

**Comment by rapenne-s** [comment]
qt5 and qt6 flavors both work for me, I've been able to load a lot of GPX at the same time, everything works for on x86_64-linux at least

**Comment by peterhoeg** [comment]
Thanks for checking @rapenne-s.

@sikmir & @womfoo as maintainers - any concerns about this? 

FYI, I have tested both qt5 and qt6 as well and both are working.

File: pkgs/applications/misc/gpxsee/default.nix [hunk 1]
```diff
@@ -19,14 +36,18 @@ stdenv.mkDerivation rec {
...
+  ] else [
+    qtbase
+    qtpositioning
+    qtsvg
+    qt5compat
+  ]);
 
   nativeBuildInputs = [ qmake qttools wrapQtAppsHook ];
 
-  preConfigure = ''
```
Comment (sikmir) [inline pkgs/applications/misc/gpxsee/default.nix:26]: Why you removed translations?

**Comment by sikmir** [comment]
11.9 is out meanwhile.

**Comment by peterhoeg** [comment]
preConfigure blows up with qt6, but just for my understanding, how can you tell that the translations are gone? Do you set the language via the LANG env var?

FYI, the qt6 issue is separate and I’ll open an issue for that.

**Comment by flokli** [comment]
Closing due to mass-ping. Please open a new PR.

**Comment by sikmir** [comment]
> preConfigure blows up with qt6, but just for my understanding, how can you tell that the translations are gone? Do you set the language via the LANG env var?

@peterhoeg Yes, if your system `LANG=en`, you can run with different locale `LANG=fi gpxsee`. Or just check `$out/share/gpxsee/translations` folder, it should contain actual translation files.

**Pull request closed without merging**